Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rust:1.59.0-alpine as builder
+RUN apk add libc-dev protoc
+
+WORKDIR /src/janus_server
+COPY janus_server/Cargo.toml /src/janus_server/Cargo.toml
+COPY janus_server/Cargo.lock /src/janus_server/Cargo.lock
+COPY janus_server/src /src/janus_server/src
+COPY db/schema.sql /src/db/schema.sql
+RUN cargo build --release --bin aggregator
+
+FROM alpine:3.15.0
+RUN apk add libgcc
```

Comment:
```suggestion
RUN apk add libgcc && apk update
```

Reply:
`apk update` fetches an updated repository index, but this would go unused twice over, because there are no subsequent `apk` commands, and the docs say:
> This command is not needed in normal operation as all applets requiring indexes will automatically refresh them after caching time expires.
Was the intent to do `apk upgrade` maybe?

Reply:
You are correct, `apk upgrade` is what we want here, or we could choose to stick with whatever is in `alpine 3.15`. Either is fine by me.

Reply:
:+1: I think I'll pass on `apk upgrade` for the time being. Dependabot will keep us up with base layer updates (speaking of, just noticed a mistake in the configuration) and this way we'll have predictable container layer sizes.